src/scss/components/_rate.scss

```diff
@@ -0,0 +1,64 @@
+$rate-color: $grey-lighter !default;
+$rate-color-active: darken($warning, 5%) !default;
+$rate-spaced: 0.25rem !default;
+
+.rate {
+  align-items: center;
+  display: flex;
+  &:not(:last-child) {
+    margin-bottom: ($block-spacing / 2);
+  }
+  &.is-spaced .rate-item:not(:last-child) {
+    margin-right: $rate-spaced;
+  }
+  &.is-disabled .rate-item {
+    cursor: initial;
+    &:hover {
+      transform: none;
+    }
+  }
+  .rate-item {
+    cursor: pointer;
+    display: inline-flex;
+    position: relative;
+    transition: all 0.3s;
+    &:hover {
+      transform: scale(1.1);
+    }
+    &.set-on .icon,
+    &.set-half .is-half {
+      color: $rate-color-active;
+    }
+    &.set-half .is-half {
+      position: absolute;
+      left: 0;
+      top: 0;
+      overflow: hidden;
+    }
+  }
+  .icon {
+    color: $rate-color;
+    line-height: 1;
+    pointer-events: none;
+    width: inherit;
+  }
+  .is-half > i {
+    position: absolute;
+    left: 0;
+  }
+  .rate-text {
+    align-items: center;
+    font-size: calc(#{$size-normal} - 2px);
+    margin-left: 6px;
+    // Alternative for size text from b-icon
+    &.is-small {
+      font-size: calc(#{$size-small} - 2px);
```

Comment:
I would try to use something along `font-size: calc(#{$size-small} / 1.5);` instead of using px for `.rate-text`